app/webroot/js/sentences_lists.menu.js

```diff
@@ -125,5 +37,94 @@ $(document).ready(function() {
         function removeFeedback(){
             $("#sentence"+sentenceId+"_saved_in_list").hide();
         }
+
+        // Clicking on "Add to list" displays the list selection.
+        // Reclicking on it hides the list selection.
+        $(".addToList").off();
+        $(".addToList").click(function(){
+            sentenceId = $(this).attr("data-sentence-id");
+            $("#addToList"+sentenceId).toggle();
+        });
+
+        // The sentence is added to the list after user has clicked
+        // the button. I was hesitating between this solution and
+        // having the sentence added to the list onChange, that is,
+        // directly after the selection in the <select>.
+        $(".validateButton").off();
+        $(".validateButton").click(function(){

```

Comment:
Had to move these back as they depend on `feedbackValid` being defined. Instead I simply `off` existing events and rebind them.